libs/atqr/domain/src/lib/valueObjects/notificationLogEntry.valueObject.ts

```diff
@@ -0,0 +1,54 @@
+import { ValueObject } from '../../utils/valueObject';
+import { Guid } from '@tokilabs/lang';
+import { NotificationChannel } from '../types/notification.types';
+export class NotificationLogEntry extends ValueObject<NotificationLogEntry> {
+  constructor(
+    public readonly id: Guid,
+    public readonly sentAt: Date,
+    public readonly notification: Notification,
+    public readonly channel: NotificationChannel,
+    public readonly to: User,
+    public readonly isSeen: boolean
+  ) {
+    super(NotificationLogEntry, [
+      'id',
+      'sentAt',
+      'notification',
+      'channel',
+      'to',
+      'isSeen',
+    ]);
+    this.id = new Guid();
```

Comment:
Do the same as we did in the Notification

Reply:
Actually, let's replace id by a hash calculated from the data both in `Notification` and `NotificationLog`